### DISCUSSION TITLE
Scalable button effect on press

### DISCUSSION THREAD

**Discussion by branikclimbs** [GeekyAnts/NativeBase#4283] [Q&A]
I would like to have a scale effect on button press similar to what these two libraries do [touchable-scale](https://github.com/JonnyBurger/touchable-scale), [react-native-pressable-scale](https://github.com/mrousavy/react-native-pressable-scale).
Is it possible? I think it is a common use case.

**Comment by intergalacticspacehighway** [comment]
@branislaav currently no, you can style button using `_pressed` prop but it won't have the transition. You can use NativeBase's [factory](https://docs.nativebase.io/nativebase-factory) with the above libraries.
